### PR TITLE
Add decode-call-result + update ACI usage

### PR DESCRIPTION
**Pull request by hanssv**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- `/decode-call-resuls` - return ACI style JSON values for a contract call result.
```
Comment (UlfNorell) [inline CHANGELOG.md]:
```suggestion
- `/decode-call-result` - return ACI style JSON values for a contract call result.
```

File: apps/aesophia_http/test/aesophia_http_SUITE.erl [hunk 2]
```diff
@@ -176,13 +186,51 @@ decode_calldata_source(_Config) ->
...
+                                    'call-result' => <<"ok">>, 'call-value' => BinValue3 }),
+    JRes4 = do_decode_call_result(#{source => SrcBin, function => <<"xyz">>,
+                                    'call-result' => <<"ok">>, 'call-value' => BinValue4 }),
+    JRes5 = do_decode_call_result(#{source => SrcBin, function => <<"abc">>,
+                                    'call-result' => <<"ok">>, 'call-value' => BinValue5 }),
+
+
+    ?assertEqual(42, JRes1),
+    ?assertEqual([<<"Hello world!">>, #{<<"One">> => [12,18]}, #{<<"Two">> => []}], JRes2),
+    ?assertEqual([#{<<"x">> => 42}, #{<<"y">> => <<"Hello world!">>}, #{<<"z">> => [[1,2],[2,3],[3,4]]}], JRes3),
```
Comment (UlfNorell) [inline apps/aesophia_http/test/aesophia_http_SUITE.erl]: ???

File: rebar.lock [hunk 3]
```diff
@@ -1,15 +1,15 @@
 {"1.1.0",
 [{<<"aebytecode">>,
   {git,"https://github.com/aeternity/aebytecode.git",
-       {ref,"2f4e1888c241a7347ffec855ab6761c2c2972f37"}},
+       {ref,"f1298870e526f4e9330447d3a281af5ef4e06e17"}},
```
Comment (lucafavatella) [inline rebar.lock]: I see this is master. Ok.

File: rebar.lock [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
   1},
  {<<"aeserialization">>,
   {git,"https://github.com/aeternity/aeserialization.git",
-       {ref,"179dd417b3c25c89d3877be88821858f91cd443c"}},
+       {ref,"3416ff5c1b4f6401bc211106063717ea1bbb1bd2"}},
```
Comment (lucafavatella) [inline rebar.lock]: I see this is master. Ok.

File: rebar.lock [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
   0},
  {<<"aesophia">>,
   {git,"https://github.com/aeternity/aesophia.git",
-       {ref,"dcae96ed21580b3b081cb955da9d8e6fd6879da1"}},
+       {ref,"758fecbb9b52fbd04138df6224a024f60788e6f2"}},
```
Comment (lucafavatella) [inline rebar.lock]: I see this is master. Ok.

**Pull request merged**